frontend/src/views/v-authorship.vue

```diff
@@ -306,27 +326,26 @@ export default {
 
     getRepoProps(repo) {
       if (repo) {
+        let files = [];
         if (this.info.isMergeGroup) {
-          // sum of all users' file type contribution
-          repo.users.forEach((author) => {
-            this.updateTotalFileTypeContribution(author.fileTypeContribution);
-          });
+          files = repo.files;
         } else {
           const author = repo.users.find((user) => user.name === this.info.author);
-          if (author) {
```

Comment:
Why can the `if (author) {` be removed? Is it guaranteed that there will be author?

Reply:
I can't quite remember now why I removed it (it may have been an oversight). In any case, a defensive author check seems like a good idea so I've added it back in.